src/cryptographic_wallets.md

```diff
@@ -18,6 +18,14 @@ If you are storing up to $1M, you MUST use a hardware wallet, such as a Ledger d
 
 If you need to store more than $1M, you MUST [contact us](./contacts.md) for specific instructions.
 
+## Clear signing
+
+You MUST ALWAYS verify, to the extent possible, the destination address(es) of any transaction you send.  Many recent attacks, from ByBit to the latest NPM poisonings, have surreptitiously replaced user specified output addresses with attacker addresses, resulting in stolen funds.  In some cases the true output addresses were displayed after alteration, in others not.
```

Comment:
We could add links to reputable hack reports, just in case.

Reply:
Done